hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -35,20 +35,18 @@
 no_gpu = not is_gpu_available()
 
 
-def get_supervised_imagenet_encoder() -> TileEncoder:
-    return ImageNetEncoder(feature_extraction_model=resnet18, tile_size=224)
+def get_supervised_imagenet_encoder_params() -> EncoderParams:
+    encoder_params = EncoderParams()
+    encoder_params.encoder_type = ImageNetEncoder.__name__
```

Comment:
You should be able to instantiate `param` classes using constructor keyword arguments, no?